R/get_gambl_metadata.R

```diff
@@ -548,10 +548,11 @@ get_gambl_metadata = function(seq_type_filter = "genome",
   
   # take `seq_type_priority` lines
```

Comment:
@lkhilton I'm curious what you think about this change.

Reply:
Was there an issue with metadata that this update is called to solve? I think should be fine, just curious whether there were any hidden bugs so we can potentially also implement something similar in data version of this function to keep outputs consistent

Reply:
This was allowing some examples of exomes and genomes from the same biopsy to come through in separate rows. This could be seen as a bug because it didn't happen systematically. It would only happen when they were given different sample_id (and this isn't always the case). When the sample_id was consistent the collapse worked as expected. The examples I was trying to deal with were MCLs with a capture and genome library with different sample_id.

Reply:
Ah I see - thanks! Ok we don't have to worry about this in the GAMBLR.data version since there are no examples yet with both capture and genome :+1:

Reply:
This is how it should have always been!!!! Can you please implement `group_by(biopsy_id, patient_id)` though - if there is ever a situation with NA or duplicate biopsy_ids (possible) you'd get unexpected results from just grouping by biopsy_id alone.